src/Compilers/CSharp/Test/Symbol/Compilation/SemanticModelGetSemanticInfoTests.cs

```diff
@@ -6715,10 +6715,6 @@ static int Main()
 ";
             var semanticInfo = GetSemanticInfoForTest<InitializerExpressionSyntax>(sourceCode);
             Assert.Null(semanticInfo.Type);
-            //Assert.Equal("int[]", semanticInfo.Type.ToString());
```

Comment:
Are these commented out because the tests don't work yet, but are expected to later? If so, I would expect a PROTOTYPE comment so that we know to uncomment them before merging.

Reply:
The commented out code, which is being deleted, was simply wrong.

---
In reply to: [171346660](https://github.com/dotnet/roslyn/pull/23921#discussion_r171346660) [](ancestors = 171346660)